### PR TITLE
ZK-4868: signature background color should be white while setBackgroundColor(null)

### PR DESCRIPTION
**Pull request by b952426**
ZK-4868: signature background color should be white while setBackgroundColor(null)

### CODE REVIEW

File: zktest/src/archive/test2/B96-ZK-4868.zul [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+	History:
+		Mon Jul 12 14:47:42 CST 2021, Created by katherine
+
+Copyright (C) 2021 Potix Corporation. All Rights Reserved.
+-->
+<zk>
+	<label multiline="true">
+		1. click button to setBackgroundColor(null).
+		2. background color shoud be white.
```
Comment (scribetw) [inline zktest/src/archive/test2/B96-ZK-4868.zul:17]: should

File: zktest/src/archive/test2/config.properties [hunk 2]
```diff
@@ -3038,6 +3038,7 @@ B90-ZK-4431.zul=A,E,Multislider
 ##manually##B96-ZK-4535.zul=A,E,Toolbar,Toolbarbutton,Combobutton
 ##manually##B96-ZK-4944.zul=A,E,Embedded,page,destory,detach
 ##manually##B96-ZK-4953.zul=A,E,Embedded,Include,defer
+##manually##B96-ZK-4868.zul=A,M,signature,backgroundColor
```
Comment (scribetw) [inline zktest/src/archive/test2/config.properties:3041]: Try to get the canvas color: https://stackoverflow.com/questions/6735470/get-pixel-color-from-canvas-on-mousemove

**Review by plijyfes — APPROVED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**